src/app/App.js

```diff
@@ -3,20 +3,22 @@ import generateQuestionForTheGameMode from '../services/game/generateQuestions';
 import { createMainMenu } from '../components/mainMenu/mainMenu';
 import createImgElementPeopleMode from '../components/recognitionImg/ImgModePeople/ImgModePeople';
 import {
-  gameMode,
-  PEOPLE_MODE_QUESTION
+    gameMode,
```

Comment:
extra indent? have you updated prettier and vsc configs according to what project has?

Reply:
Yes, I installed prettier. It change code that way :/

Reply:
I see that prettier has 2 tab rule, make sure that this is fine